code/modules/jobs/job_types/roguetown/adventurer/types/migrants/heartfelt/heartfeltlord.dm

```diff
@@ -77,6 +82,221 @@
+
+/datum/outfit/job/heartfelt/lord/chief/pre_equip(mob/living/carbon/human/H)
+	..()
+	head = /obj/item/clothing/head/roguetown/helmet/sallet/beastskull
+	shirt = /obj/item/clothing/suit/roguetown/armor/gambeson/heavy
+	belt = /obj/item/storage/belt/rogue/leather/battleskirt/faulds
+	shoes = /obj/item/clothing/shoes/roguetown/boots/leather/reinforced
+	pants = /obj/item/clothing/under/roguetown/heavy_leather_pants
+	cloak = /obj/item/clothing/cloak/darkcloak/minotaur/red
+	armor = /obj/item/clothing/suit/roguetown/armor/brigandine/heartfelt
+	neck = /obj/item/clothing/neck/roguetown/leather
+	beltr = /obj/item/rogueweapon/huntingknife
+	gloves = /obj/item/clothing/gloves/roguetown/leather/black
+	backl = /obj/item/storage/backpack/rogue/satchel/heartfelt // Paper and Feather
+	backpack_contents = list(
+		/obj/item/rogueweapon/huntingknife/idagger/steel/special = 1,
+		/obj/item/rogueweapon/scabbard/sheath = 1,
+		/obj/item/storage/belt/rogue/pouch/coins/rich = 1,)
+	id = /obj/item/scomstone
+	if(H.mind)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/self/convertrole/heartfelt)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/order/heartfelt/retreat)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/order/heartfelt/bolster)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/order/heartfelt/charge)
+		H.mind.AddSpell(new /obj/effect/proc_holder/spell/invoked/order/heartfelt/forheartfelt)
+		// H.mind.AddSpell(new/obj/effect/proc_holder/spell/invoked/order/heartfelt/focustarget)
+		H.verbs |= list(/mob/living/carbon/human/mind/proc/setordersheartfelt)
+	H.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/maces, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/axes, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/bows, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/unarmed, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/swords, 3, TRUE)
+	H.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/reading, 4, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/sneaking, 1, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
+	H.adjust_skillrank(/datum/skill/misc/riding, 3, TRUE)
+	H.change_stat("strength", 3)
+	H.change_stat("endurance", 3)
+	H.change_stat("speed", 1)
+	H.change_stat("constitution", 2)
+	H.change_stat("perception", -2)
+	H.change_stat("intelligence", -1)
+	H.change_stat("fortune", 5)
+	ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
+	var/weapons = list("Tetsubo", "Double-head Greataxe", "Great Mace + Shield", "Battle Axe + Shield", , "Warhammer + Shield")
```

Comment:
The tetsubo is the mace equivalent of a minotaur greataxe and should remain unique to its own class. It makes no sense to make it available to the Lord of Heartfelt.

The great mace is two-handed and is barely functional held in one hand, so I am not sure why it is paired with a shield.